CONTRIBUTING.md

```diff
@@ -55,7 +55,22 @@ Since GUAC is a complex project, there are 5 topic areas of interest:
 - Front-end: GUAC visualisation libraries, Javascript, CSS
 - Clients: GUAC CLI, CI Checks, Policy Engines, IDE Plugins, etc.
 
-The contribution ladder is summarized in the table below:
+In addition, there are 3 non-code topic areas:
+
+- Triage & project management: Contributors who help with managing the issue trackers and other project management activities (like assigning issues to milestones).
```

Comment:
Yeah, I'm not sure what it would look like at this point. The reason I included it here is because it's a still topic area, and the ladder is special cased below. For that reason, I think it makes sense as presented, but I'm open to suggestions for alternative wording/presentation. On the other hand...

Full disclosure: I included this now because It's an area where I can help. It's not something that most folks are clamoring to help with in any project Iv'e seen, so it might be that I'm the only one who ends up with this role for a while anyway. Another approach is to leave this out and either 1. I don't contribute in this area or 2. the maintainers grant one-off triage privileges on the repos to help with this and we treat future requests on a one-off basis until we start getting a significant number of those.